index.d.ts

```diff
@@ -41,12 +41,12 @@ interface MultipartValue<T> {
 }
 
 interface MultipartErrors {
-    PartsLimitError: FastifyError,
-    FilesLimitError: FastifyError,
-    FieldsLimitError: FastifyError,
-    PrototypeViolationError: FastifyError,
-    InvalidMultipartContentTypeError: FastifyError,
-    RequestFileTooLargeError: FastifyError
+    PartsLimitError: FastifyErrorConstructor,
```

Comment:
This is the wrong fixing approach. All the `MultiparErrors` is not construable. They should be the instance of `FastifyError` or `Error`.